contracts/pools/PoolCollection.sol

```diff
@@ -595,6 +610,17 @@ contract PoolCollection is IPoolCollection, Owned, BlockNumber, Utils {
         _resetTradingLiquidity(bytes32(0), pool, data, data.liquidity, TRADING_STATUS_UPDATE_ADMIN);
     }
 
+    /**
+     * @inheritdoc IPoolCollection
+     */
+    function disableTradingByNetwork(Token pool) external only(address(_network)) {
```

Comment:
this skips the entire logic associated with disabling trading on pools - reducing trading liquidity, emitting the associated events etc.
given these implications, I think the better approach would be to allow the network to execute the current enableTrading method

Reply:
Fixed